org.dpsoftware.FireflyLuciferin.appdata.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.dpsoftware.FireflyLuciferin</id>
+    <provides><id>org.dpsoftware.FireflyLuciferin.desktop</id></provides>
+    <launchable type="desktop-id">org.dpsoftware.FireflyLuciferin.desktop</launchable>
+    <name>Firefly Luciferin</name>
+    <developer_name>Firefly Luciferin</developer_name>
+    <summary>Ambilight clone</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/sblantipodi/firefly_luciferin?tab=GPL-3.0-1-ov-file</project_license>
```

Comment:
GPL-3.0 is NOT proprietary. So is it proprietary or not?

Reply:
fixed, thanks